components/core/NewWebsitePrototypeHeader.js

```diff
@@ -261,8 +257,7 @@ export const NewWebsitePrototypeHeader = (props) => {
           <div css={open ? STYLES_BURGER_BUN2_OPEN : STYLES_BURGER_BUN2} />
         </div>
         <div open={open} css={open ? STYLES_MENU_OPEN : STYLES_MENU}>
-          <a href={designSystemURL}>Design system</a>
-          <a href={viewSourceURL}>View source</a>
+          <a href={commuityURL}>Get Involved</a>
```

Comment:
"Sign in"
"Get involved"

Its not a title

Reply:
got it will change all the non-tiles